app/services/databaseService.js

```diff
@@ -46,4 +46,34 @@ async function getPaymentData (searchString = '', limit = 20, offset = 0) {
   }
 }
 
-module.exports = { getPaymentData, PaymentDataModel }
+// payment details API
+const PaymentDetailModel = sequelize.define('payment_activity_data', {
```

Comment:
Given we are on a tight deadline this does not need to be addressed right now but I think it would be better to define the models in a separate file.